### DISCUSSION TITLE
Banco de dados

### DISCUSSION THREAD

**Discussion by ivannaprata** [EasyRoomCompany/Guardian#68] [Ideas]
@fabianopmelo as informações que você pontou no WhatsApp sobre a tabela tipo_de_sala são bem pertinentes e precisamos criar-lá para posteriormente fazer os inserts. 

Assim que o @kevinDsousa criar o código das rotas, poderemos inserir dados e fazermos uma prévia de análise estatística.

**Comment by kevinDsousa** [comment]
Boa noite @ivannaprata Estou finalizando aqui as rotas com todas as inserções para que o CRUD funcione, quando concluírmos vamos precisar atrelar a interface. Quando concluímos podemos inserir os dados que ficarão para a aplicação. Peço que tenham ja em mente todos os dados para que possamos ganhar tempo.

**Comment by ivannaprata** [comment]
@kevinDsousa estou revisando o código das tabelas para que melhorias possam ser aplicadas nos tipos de dados dos campos. Vou te mandar aqui a sugestão.

**Comment by kevinDsousa** [comment]
@ivannaprata Boa tarde, ficarei no aguardo par aque possamos atualizar no sistema

**Comment by kevinDsousa** [comment]
Proxima talk vamos fazer uma analise no banco de dados atual e tentar identificar as entidades para um bom funcionamento do software, peço a todos que ja começem a pensar nas possiveis entidades que vamos usar.
